### ISSUE TITLE
Add Wikidata IDs for mythological and historical characters in API output

### ISSUE THREAD

**Issue by lehkost** [dracor-org/dracor-api#143]
Since we added Wikidata IDs for mythological and historical characters in ```<particDesc>```, we can provide these numbers in the following API outputs:
* /play/{playname}
* /play/{playname}/cast

An example for how the information is stored in TEI is here (taken from [Alfieri's "Antigone"](https://dracor.org/id/ita000003)):
```xml
<particDesc>
  <listPerson>
  […]
  <person xml:id="antigone" sex="FEMALE" ana="http://www.wikidata.org/entity/Q131351">
    <persName>Antigone</persName>
  </person>
  <person xml:id="creonte" sex="MALE" ana="http://www.wikidata.org/entity/Q732066">
    <persName>Creonte</persName>
  </person>
  […]
</listPerson>
```

**Comment by cmil** [comment]
@lehkost This is done, isn't it?

**Comment by lehkost** [comment]
I think so, can be closed, next stop #144. ☺️
